presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -1445,22 +1485,27 @@ public void testMaterializedViewForJoin()
             queryRunner.execute(format("CREATE TABLE %s WITH (partitioned_by = ARRAY['ds']) AS " +
                     "SELECT orderkey, '2020-01-01' as ds FROM orders WHERE orderkey < 1000 " +
                     "UNION ALL " +
-                    "SELECT orderkey, '2019-01-02' as ds FROM orders WHERE orderkey > 1000", table1));
+                    "SELECT orderkey, '2019-01-02' as ds FROM orders WHERE orderkey > 1000 and orderkey < 2000", table1));
```

Comment:
why we need to add `orderkey < 2000`?

Reply:
Running this unit test locally is stuck there and then crashes, due to too large table loaded into memory I guess. So adding < 2000 here to limit # of records. Correctness wise, it doesn't matter whether or not we have < 2000 here.